src/extension/devtools/devtools.js

```diff
@@ -1,72 +1,67 @@
-// this script is called whenever a user opens the chrome devtools
-// on a page. We check to see if Apollo Client exists, and if
-// it does, then we create the Apollo devtools, otherwise we poll each
-// second to see if its been created
-//
-// XXX we should show a better loading state here instead of nothing
-// much like a connector / loading page while data is loaded. Then
-// after a timeout, we can show instructions / docs for setting up
-// AC to work with the devtools
+import Relay from '../../Relay';
+import { 
+  DEVTOOLS_INITIALIZED, 
+  CREATE_DEVTOOLS_PANEL,
+  ACTION_HOOK_FIRED,
+} from '../constants';
 
-let panelLoaded = false;
-let panelCreated = false;
-let panelShown = false;
-// stop after 10 seconds
-let checkCount = 0;
-let loadCheckInterval;
+const inspectedTabId = chrome.devtools.inspectedWindow.tabId;
+const devtools = new Relay('devtools');
```

Comment:
It seems a bit confusing to have both `window.devtools` and a local variable called `devtools`. That said, I think/hope we can get rid of `window.devtools` (see other comments), so this should sort itself out?